### PR TITLE
Animation added to art directory

### PR DESCRIPTION
**Pull request by Aryan741x**
Please combine this PR with Hacktoberfest. This general menu bar animation is meant to aid in others' viewing and understanding of it.

### CODE REVIEW

**Comment by LaurelineP** [comment]
Thanks for your contribution 🙌
Before we could merge your contribution, the following feedbacks need to be addressed first.
Please address the given feedback(s)

# Feedbacks  
- [ ] add modifications in `include.js` file
- [ ] remove the unused pictural reprensentation file not used in the details you will provide

## Acceptance criteria for a contribution
- [x] the contribution includes a new folder named after the author github user name  in `./Art`
- [x] the contribution includes an HTML file
    - [x] no CSS
    - [x] no JS
    - [x] no embedded  SVG 
    - [x] has a `<link>` tag to the CSS file
- [x] the contribution includes a CSS file
- [x] the contribution includes the animated file (gif) or picture (jpeg, jpg, png )
    _the pictural representation must not include the browser frame, just the animation_
- [ ] the contribution includes the correct contribution's details in `include.js`

Please address the above feedbacks and/or unchecked acceptance criteria - if you need guidance , please refer to the `README.md` file.

**Comment by Aryan741x** [comment]
@LaurelineP ok I will change and send the pull request

**Comment by Aryan741x** [comment]
@LaurelineP Kindly Merge this pr. As a novice on GitHub, I have made every effort to address the merge issue and completed the necessary

File: Art/Aryan741x/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <title>DAY 2 Challenge</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="menu no-animation">
+        <div></div>
+        <div></div>
+        <div></div>
+    </div>
+    <script src="main.js"></script>
```
Comment (LaurelineP) [inline Art/Aryan741x/index.html:15]: # Feedback - remove js script tag
Please remove the js script tag

**Comment by Aryan741x** [comment]
@LaurelineP index.html resolved

**Pull request merged**